Dockerfile

```diff
@@ -13,19 +13,31 @@ RUN dnf install -y gcc git make xz-devel
 WORKDIR /tmp
 
 RUN git clone https://github.com/ipxe/ipxe.git && \
-      cd ipxe && \
-      git reset --hard $IPXE_COMMIT_HASH && \
-      cd src && \
-      ARCH=$(uname -m | sed 's/aarch/arm/') && \
-      # NOTE(elfosardo): warning should not be treated as errors by default
-      NO_WERROR=1 make bin/undionly.kpxe "bin-$ARCH-efi/snponly.efi"
+     cd ipxe && \
+     git reset --hard $IPXE_COMMIT_HASH && \
+     cd src && \
+     ARCH=$(uname -m | sed 's/aarch/arm/') && \
+     # NOTE(elfosardo): warning should not be treated as errors by default
+     NO_WERROR=1 make bin/undionly.kpxe "bin-$ARCH-efi/snponly.efi"
 
 COPY prepare-efi.sh /bin/
 RUN prepare-efi.sh centos
 
+# build actual image
 FROM $BASE_IMAGE
 
-ENV PKGS_LIST=main-packages-list.txt
+LABEL io.k8s.description="Container image to run OpenStack Ironic as part of Metal³"
```

Comment:
In BMO we also do `io.k8s.*` labels, do we want to add them too, or take them away from BMO and replace them with org.opencontainers ones?

Reply:
I would leave org.opencontainers only, unles we have a strong reason to keep io.k8s too

Reply:
We've had the k8s one in BMO since 2020 by @dhellmann and it is actually also copied now to IRSO by @dtantsur . Googling around it seems to be mostly OpenShift thing: https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/images/creating-images#defining-image-metadata

Reply:
+1 for skipping the io.k8s labels. If they are needed for openshift I am fine with keeping them though

Reply:
I don't think we actually care in OpenShift too much. If we discover we need them, we can always re-add in our downstream fork.